d2l-scroll-wrapper.html

```diff
@@ -125,6 +125,7 @@
 
 			attached: function() {
 				this.async(this.checkScrollbar);
+				this.async(this.checkScrollbar);
```

Comment:
For some reason, the scrollWidth/offsetWidth doesn't update correctly until after the scroll-left/scroll-right attributes are updated. Probably triggering a rerender. Any other ideas?

I also tried #52, but @rlanger had an issue with it.